hashes/src/util.rs

```diff
@@ -113,7 +121,7 @@ macro_rules! hash_newtype {
         #[repr(transparent)]
         pub struct $newtype($hash);
 
-        $crate::hex_fmt_impl!($newtype);
+        $crate::hex_fmt_impl!($reverse, $newtype);
```

Comment:
Can we have `$reverse` _after_ `$newtype` so the args are ordered the same as for other macros.

Reply:
Generics have to be at the end and I think it'd be a bit weird to have `$reverse` between type name and generics. Agree this one is not great either. :man_shrugging:

Reply:
What if we had a separate macro `lpmi_tmf_xeh` which was the reversed version, rather than having a parameter for `$reverse`.

*ducks*